src/index.html

```diff
@@ -2,12 +2,217 @@
+            <img src="images/yizha.png" alt="" class="products__img">
+            <div class="products">
+              <h2 class="products__place">YIZHA</h2>
+              <p class="products__list">₴₴ • Американська • Бургери • Вулична їжа</p>
+              <p class="products__time">35 - 45 Min</p>
+            </div>
+          </div>
+          <div class="catalog__item products">
+            <img src="images/musafir.png" alt="" class="products__img">
+            <div class="products">
+              <h2 class="products__place">Musafir (вул. Саксаганського)</h2>
+              <p class="products__list">₴₴ • Близькосхідна</p>
+              <p class="products__time">30 - 40 Min</p>
+            </div>
+          </div>
+          <div class="catalog__item products">
+            <img src="images/wokwei2.png" alt="" class="products__img">
+            <div class="products">
+              <h2 class="products__place">WOKWEI</h2>
+              <p class="products__list">₴ • Китайська • Азіатська</p>
+              <p class="products__time">40 - 50 Min</p>
+            </div>
+          </div>
+          <div class="catalog__item products">
+            <img src="images/milkBar.png" alt="" class="products__img">
+            <div class="products">
+              <h2 class="products__place">Milk Bar</h2>
+              <p class="products__list">₴₴ • Десерти • Європейська</p>
+              <p class="products__time">25 - 35 Min</p>
+            </div>
+          </div>
+          <div class="catalog__item products">
+            <img src="images/mimosa.png" alt="" class="products__img">
+            <div class="products">
+              <h2 class="products__place">Mimosa Brooklyn Pizza</h2>
+              <p class="products__list">₴₴ • Італійська • Піца • Американська</p>
+              <p class="products__time">15 - 25 Min</p>
+            </div>
+          </div>
+          <div class="catalog__item products">
+            <img src="images/toBe.png" alt="" class="products__img">
+            <div class="products">
+              <h2 class="products__place">To Be</h2>
+              <p class="products__list">₴₴ • Японська • Азіатська • Суші</p>
+              <p class="products__time">25 - 35 Min</p>
+            </div>
+          </div>
+          <div class="catalog__item products">
+            <img src="images/druzi.png" alt="" class="products__img">
+            <div class="products">
+              <h2 class="products__place">Druzi Cafe (вул. Прорізна)</h2>
+              <p class="products__list">₴₴ • Американська • Сучасна європейська</p>
+              <p class="products__time">30 - 40 Min</p>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="footer">
+      <div class="footer-logo">
+        <div class="footer-logo__container">
+          <a href="#" class="footer-logo__link"><img src="images/white-logo.svg" alt="UBER Eats" class="logo__img"></a>
```

Comment:
Делай переносы для улучшения читаемости
```suggestion
          <a href="#" class="footer-logo__link">
            <img src="images/white-logo.svg" alt="UBER Eats" class="logo__img">
          </a>
```